<launch>
    <arg name="botID"/>
    <group ns="rescue_agents">
        <node pkg="rescue_center" type="rescue_agent_node.py" name="rescue_agent_$(arg botID)" output="screen">
          <param name="~distressed_veh" type="string" value="autobot$(arg botID)" />
          <param name="~everythingOK" type="bool" value="false" />
          <param name="~velocity_backwards" type="double" value="-0.1"/>
          <param name="~demo_mode" type="bool" value="false"/>
        </node>
    </group>
</launch>   